examples/twittermap/web/conf/application.conf

```diff
@@ -105,11 +105,11 @@ endDate = "2017-05-01T00:00:00.000"
 # cacheThreshold = 30000
 
 # for the debug purpose, we can load a smaller json to speed up the front-end
-us.city.path = "/public/data/city.sample.json"
+# us.city.path = "/public/data/city.sample.json"
```

Comment:
Please do not submit changes to application.conf, since it has nothing to do with your PR.